src/components/ViewBusiness/ViewBusiness.jsx

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable react/prop-types */
+import { useEffect, useState } from 'react';
+import {
+  ChakraProvider,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Flex,
+  Avatar,
+  Box,
+  Heading,
+  IconButton,
+  Text,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Td,
+  TableContainer,
+  HStack,
+  Tag,
+  Stack,
+  Divider,
+} from '@chakra-ui/react';
+import { DeleteIcon, EditIcon, DownloadIcon, CalendarIcon } from '@chakra-ui/icons';
+import { useBackend } from '../../contexts/BackendContext';
+import kevinLiu from './kevinLiu.jpg';
+
+const ViewBusiness = ({ id }) => {
+  const [business, setBusiness] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+  const [data, setData] = useState([]);
+
+  const { backend } = useBackend();
+
+  const fetchBusiness = async () => {
+    try {
+      console.log(`Fetching business details for ID: ${id}`);
+      const response = await backend.get(`/business/${id}`);
+      setData(response.data[0]);
+      console.log('Response received:', response);
+
+      setBusiness({
+        name: response.data.Name,
+        email: response.data.email,
+        website: response.data.Website,
+        location: response.data.location,
+      });
+    } catch (error) {
+      console.error('Error:', error);
+      setError(error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  // ViewBusiness.PropTypes = {
+  //   id: PropTypes.number.isRequired,
+  // }
+
+  useEffect(() => {
+    if (id) {
+      fetchBusiness();
+    }
+  }, [id]);
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  }
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  return (
```

Comment:
in the future try to break down the jsx blocks into smaller functions that return jsx for readability